clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -134,6 +134,10 @@ Changes in existing checks
   <clang-tidy/checks/bugprone/unused-local-non-trivial-variable>` check by
   ignoring local variable with ``[maybe_unused]`` attribute.
 
+- Fixed :doc:`cppcoreguidelines-missing-std-forward
+  <clang-tidy/checks/cppcoreguidelines/missing-std-forward>` check giving false
+  positives for deleted functions.
+
```

Comment:
Nit: I think the release note should follow the convention of other changes by starting with `Improved` (I realize the release note after this is not doing that, but every other check is).

Reply:
Sure looks more consistent :)